src/classes/UTIL_SortContacts_TEST.cls

```diff
@@ -0,0 +1,193 @@
+        contacts[4].LastName = 'Brett';
+        contacts[5].FirstName = 'Charles';
+        contacts[5].LastName = 'Brett';
+        contacts[6].FirstName = 'Cat';
+        contacts[6].LastName = 'Pierre';
+        contacts[7].FirstName = 'Lyla';
+        contacts[7].LastName = 'Pierre';
+        contacts[8].FirstName = 'Annie';
+        contacts[8].LastName = 'Pierre';
+
+        Test.startTest();
+        utilSortCons.add(new UTIL_SortContacts(contacts[0]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[1]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[2]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[3]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[4]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[5]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[6]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[7]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[8]));
+        Test.stopTest();
+        utilSortCons.sort();
+
+        List<Contact> sortedContacts = new List<Contact>();
+        for (UTIL_SortContacts us : utilSortCons) {
+            sortedContacts.add(us.contactRecord);
+        }
+
+        System.assertEquals(contacts[4], sortedContacts[0]);
+        System.assertEquals(contacts[5], sortedContacts[1]);
+        System.assertEquals(contacts[3], sortedContacts[2]);
+        System.assertEquals(contacts[8], sortedContacts[3]);
+        System.assertEquals(contacts[6], sortedContacts[4]);
+        System.assertEquals(contacts[7], sortedContacts[5]);
+        System.assertEquals(contacts[2], sortedContacts[6]);
+        System.assertEquals(contacts[0], sortedContacts[7]);
+        System.assertEquals(contacts[1], sortedContacts[8]);
+
+    }
+
+    /*********************************************************************************************************
+    * @description Test Method to test sorting Contacts alphabetically by LastName only.
+    */
+    @isTest
+    private static void sortContactsLastName() {
+        List<UTIL_SortContacts> utilSortCons = new List<UTIL_SortContacts>();
+        List<Contact> contacts = UTIL_UnitTestData_TEST.getMultipleTestContacts(3);
+
+        contacts[0].FirstName = 'Will';
+        contacts[0].LastName = 'Zap';
+        contacts[1].FirstName = 'William';
+        contacts[1].LastName = 'Brett';
+        contacts[2].FirstName = 'Abigail';
+        contacts[2].LastName = 'Pierre';
+
+
+        Test.startTest();
+        utilSortCons.add(new UTIL_SortContacts(contacts[0]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[1]));
+        utilSortCons.add(new UTIL_SortContacts(contacts[2]));
+        Test.stopTest();
+        utilSortCons.sort();
+
+        List<Contact> sortedContacts = new List<Contact>();
+        for (UTIL_SortContacts us : utilSortCons) {
+            sortedContacts.add(us.contactRecord);
+        }
+
+        System.assertEquals(contacts[1], sortedContacts[0]);
```

Comment:
this scared me at first but then I remember we are sorting :) so we can depend on order *phew*